examples/apps/quick_tap.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Quick Tap - tap your cube as fast as possible when the colors match, but never tap on red!
+
+After you and Cozmo select your cubes, the third cube acts as the countdown for each round.
+
+'''
+import asyncio, random, sys, time
+
+import cozmo
+
+from cozmo.lights import blue_light, Color, green_light, Light, red_light, white_light
+from cozmo.util import degrees, distance_mm, radians, speed_mmps
+
+yellow_light = Light(Color(name = 'yellow', rgb = (255, 255, 0)))
+purple_light = Light(Color(name = 'purple', rgb = (255, 0, 255)))
+
+LIGHT_COLORS_LIST = [red_light, blue_light, green_light, yellow_light, purple_light, white_light]
+
+CHOOSE_CUBES_STATE = 'choose_cubes'
+GAME_STATE = 'game'
+
+BUZZER_DIFFERENT = 'buzzer_different'
+BUZZER_RED = 'buzzer_red'
+BUZZER_SAME = 'buzzer_same'
+
+RATE_BUZZER_DIFFERENT = 0.17
+RATE_BUZZER_RED = 0.33
+
+RATE_COZMO_ACCURACY = 0.9
```

Comment:
I think a comment about how this var is used would be good, and that the user can make the game easier presumably by decreasing this value?